card-data/01-character-upbringing.yaml

```
@@ -0,0 +1,264 @@
+      - Navigate a physical or social labyrinth
+      - Gauge someone's wants or needs
+  back:
+    <<: *tback
+    name: City
+    desc: >
+      //"...For the right price." -- Basler
+
+      Not everything can be reduced to a commercial transaction,
+      but city dwellers sometimes try anyway.
+      Some of what you've seen has made you numb.
+      Other times, you've decided rightly or wrongly
+      that you can't help, and have moved on anyway.
+    prompts:
+      - Compromise a principle or ideal for expedience
+      - Overcomplicate a plan or solution
+      - Look past pain or hurt rather than soothe it
+- front: &Country
+    <<: *tfront
+    name: Country
+    desc: >
+      //"Take a deep breath of that country air!" -- Tana
+
+      You grew up among farmers, ranchers, or other folk who lived
+      off the land. You might have lived in a tight-knit community,
+      or been miles away from the nearest neighbor.
+
+      You're familiar with the plants or animals of your region.
+      You're used to putting in an honest day's work.
+
+      You've learned to approach situations via effort.
+    prompts:
+      - Endure wearying labor, physical or emotional
+      - Devise practical solutions to hard problems
+      - Understand someone's unspoken hardships
+  back:
+    <<: *tback
+    name: Country
+    desc: >
+      //"Smells like cow breakfast" -- Basler
+
+      Not everyone appreciates the necessity of hard work.
+      But sometimes, just plowing through isn't the right answer either.
+      You're used to getting down in the mud,
+      but it's important to reach for the stars as well.
+    prompts:
+      - Condemn someone for shunning hard work
+      - Balk at complex explanations or solutions
+      - Sacrifice tomorrow's hopes for today's needs
+- front: &Frontier
+    <<: *tfront
+    name: Frontier
+    desc: >
+      //"The forest demands self-sufficiency" -- Woody
+
+      You grew up at the edges of civilization, whether a harsh
+      desert, a lonely mountain, a deep forest, or somewhere
+      equally remote. You learned to fend for yourself from
+      a young age.
+
+      You can hunt, trap, or otherwise obtain your basic necessities.
+      Your life is a careful balance of trust and caution.
+
+      You've learned to approach situations through resourcefulness.
+    prompts:
+      - Come well-prepared for harsh conditions
+      - Spot what you need in the environment
+      - Anticipate danger
+  back:
+    <<: *tback
+    name: Frontier
+    desc: >
+      //"Well the forest sounds pretty demanding" -- Basler
+
+      When much of your life is preoccupied with taking care
+      of yourself and your needs, it's easy to lose sight of
+      others' wants.
+      Carefulness and selfishness are matters of degree.
+    prompts:
+      - Take something others need later for what you need now
+      - Cross a line in pursuit of your goals
+      - Imagine a threat in every shadow
```

```suggestion
      - Prioritize your immediate needs over the plans or stores of others
      - Disregard the scruples and niceties of civilized folk
      - Imagine a threat in every shadow
```
These are hard concepts to try to boil down. I've had a go...